src/env.c

```diff
@@ -1,6 +1,26 @@
 #include "rlang.h"
 
+
+bool r_is_unbound_value(SEXP x) {
+  return x == R_UnboundValue;
+}
+
 SEXP r_mut_env_parent(SEXP env, SEXP new_parent) {
   SET_ENCLOS(env, new_parent);
   return env;
 }
+
+bool r_is_env(SEXP x) {
+  return r_typeof(x) == ENVSXP;
+}
+
+SEXP r_env_get(SEXP env, SEXP sym) {
+  return Rf_findVarInFrame3(env, sym, TRUE);
+}
+
+SEXP r_ns_env(const char* pkg) {
```

Comment:
I guess, but the lack of a verb in those function names leaves me guessing as to what they do.